salt/api-gateway/config/opt-kong-docker-compose.yaml

```diff
@@ -23,3 +23,4 @@ services:
             # directories (host:container)
             # these paths are specified in `loris2.conf`
             - /var/log/kong:/var/log/kong
+            - /usr/local/kong/logs:/usr/local/kong/logs
```

Comment:
Minor suggestion moving forward, I like to suffix the permissions on volume mounts, e.g. `/usr/local/kong/logs:/usr/local/kong/logs:rw` or `/opt/kong/kong.conf:/etc/kong/kong.conf:ro`. Find it helpful when looking at compose files and trying to figure out the purpose of the volume mount.